query/mutation.go

```diff
@@ -286,6 +286,15 @@ func checkIfDeletingAclOperation(ctx context.Context, edges []*pb.DirectedEdge)
 
 	isDeleteAclOperation := false
 	for _, edge := range edges {
+		unReservedPredicate := false
```

Comment:
we should only look at edge.Predicate here

Reply:
Predicate is not a member of DirectedEdge{}